arangod/RocksDBEngine/RocksDBIterators.h

```diff
@@ -26,7 +26,7 @@
 
 #include "Indexes/Index.h"
 #include "Indexes/IndexIterator.h"
-#include "RocksDBEngine/RocksDBColumnFamily.h"
+#include "RocksDBEngine/RocksDBColumnFamilyManager.h"
```

Comment:
```suggestion
```
It seems that this header is not used here since there are not other changes. Perhaps other files that include `RocksDBIterators.h` depend on it implicitly, but in that case these files should include `RocksDBColumnFamilyManager.h` directly.